packages/paste-core/primitives/box/__tests__/StyleFunctions.test.tsx

```diff
@@ -171,4 +171,27 @@ describe('getCustomElementStyles', () => {
       });
     }
   });
+
+  it('should handle variants that are not present on the theme', () => {
```

Comment:
NIT: For readability could this include a brief description of how it should handle variants that are not present? For example, does the function return defaults for variants that do not match the theme? Does it throw an error?

Reply:
Updated, and threw in a more explicit test to expect that it does not throw too. Good suggestion